tests/pages/IdentityOverviewPage.test.tsx

```diff
@@ -42,409 +15,444 @@ jest.mock("@src/dbClient", () => {
   };
 });
 
+function makeAsPrimaryRegExFactory(identity: Identity): RegExp {
+  return new RegExp(
+    "Make " + identity.value + " my primary " + identity.type.toLowerCase(),
+  );
+}
+
+const userId = "ac3f358d-d2c9-487e-8387-2e6866b853c9";
+
+const nonPrimaryEmailIdentity: Identity = {
+  id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
+  primary: false,
+  status: "validated",
+  type: IdentityTypes.EMAIL,
+  value: "Test@test.test",
+};
+
+const primaryEmailIdentity: Identity = {
+  id: "6tf443c1-530b-4982-878d-33f0def6a7cf",
+  primary: true,
+  status: "validated",
+  type: IdentityTypes.EMAIL,
+  value: "test4@test.test",
+};
+
+const unvalidatedEmailIdentity: Identity = {
+  id: "77yt43c1-530b-4982-878d-33f0def6a7cf",
+  primary: false,
+  status: "unvalidated",
+  type: IdentityTypes.EMAIL,
+  value: "test6@test.test",
+};
+
+const nonPrimaryPhoneIdentity: Identity = {
+  id: "81z343c1-530b-4982-878d-33f0def6a7cf",
+  primary: false,
+  status: "validated",
+  type: IdentityTypes.PHONE,
+  value: "0642424242",
+};
+
+const primaryPhoneIdentity: Identity = {
+  id: "81z343c1-530b-4982-878d-33f0def6a7cf",
+  primary: true,
+  status: "validated",
+  type: IdentityTypes.PHONE,
+  value: "0642424242",
+};
+
+const unvalidatedPhoneIdentity: Identity = {
+  id: "81z343c1-530b-4982-878d-33f0def6a7cf",
+  primary: false,
+  status: "unvalidated",
+  type: IdentityTypes.PHONE,
+  value: "0642424242",
+};
+
 describe("IdentityOverviewPage", () => {
-  beforeEach(() => {
-    fetch.resetMocks();
-  });
+  describe("Identity type: EMAIL", () => {
+    it("should render proper email breadcrumbs", () => {
+      render(
+        <IdentityOverviewPage
+          identity={nonPrimaryEmailIdentity}
+          userId={userId}
+        />,
+      );
 
-  const userId = "ac3f358d-d2c9-487e-8387-2e6866b853c9";
+      expect(screen.getByText("Email address")).toBeInTheDocument();
+    });
 
-  const nonPrimaryIdentity: Identity = {
-    id: "8f79dcc1-530b-4982-878d-33f0def6a7cf",
-    primary: false,
-    status: "validated",
-    type: IdentityTypes.EMAIL,
-    value: "test@test.test",
-  };
+    it("should display the relevant options delete & mark as primary buttons but not validate identity one for a non primary validated email adress", () => {
```

Comment:
```suggestion
    it("should display the relevant options delete & mark as primary buttons but not validate identity one for a non primary validated email address", () => {
```

I find this really confusing. You also call the delete button "option delete" in the various tests. Why not call it delete button? :)